packages/editor/src/components/block-list/style.scss

```diff
@@ -368,6 +369,11 @@
 		}
 	}
 
+	// Center
+	&[data-align="center"] {
```

Comment:
Have you considered doing `&[data-align] [data-align="center"]` to be a bit more specific to situations in where it's following another floated item?

Reply:
This is a good idea, however I would expect my block to not float no matter what it follows.

Reply:
yeah, I have no idea what I meant there. I tested it locally and I think this is good to go.